customisation/translation.mdx

```diff
@@ -6,6 +6,16 @@ Translation files are located in the `.chainlit/translations` directory. The fil
 
 <Note>The language is dynamically set for each user based on the language of the browser. The default language is `en-US`.</Note>
 
+## Customizing UI Text
+
+In addition to standard translations, you can customize the names of front-end components, molecules, and organisms used within the UI. Each UI element is associated with a unique translation key in the translation files. By modifying these keys, you can personalize or localize the UI text according to your needs.
```

Comment:
I think we shouldn't share the name molecules and organisms because it's mostly an internal way to classify the components.

```suggestion
In addition to standard translations, you can customize the names of front-end components used within the UI. Each UI element is associated with a unique translation key in the translation files. By modifying these keys, you can personalize or localize the UI text according to your needs.
```